utils/config.go

```diff
@@ -553,6 +553,8 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	hasImageProxy := c.ServiceSettings.ImageProxyType != nil && *c.ServiceSettings.ImageProxyType != "" && c.ServiceSettings.ImageProxyURL != nil && *c.ServiceSettings.ImageProxyURL != ""
 	props["HasImageProxy"] = strconv.FormatBool(hasImageProxy)
 
+	props["RunJobs"] = strconv.FormatBool(*c.JobSettings.RunJobs)
```

Comment:
Expose this to the clients can be confusing, because you can receive RunJobs true when you have a api that acts as a worker, and false when you have a api that doesn't act as a worker, and in both cases you are running the jobs, but sometimes not in the api node.

Reply:
However, it's not claiming to be anything other than the raw, dumb config value on that server and is interpreted smartly on the client. 

I think your point (a good one) highlights the need for a distributed config solution (like etcd) that would make the interpretation of values like this require less knowledge on the client (as well as other advantages)

Otherwise we would need a new API endpoint to expose a value that is more intelligent.